package.json

```
@@ -3,18 +3,19 @@
   "version": "1.0.0",
   "description": "",
   "main": "index.js",
+  
   "scripts": {
     "test": "jest",
-    "lint": "eslint '*.js'",
+    "lint": "npm run lint:js && npm run lint:html",
+    "lint:js": "eslint '*.js'",
     "lint:fix": "eslint '*.js' --fix",
```

```suggestion
    "lint:js-fix": "eslint '*.js' --fix",
```
Vielleicht könnten wir durch diese Änderung klarmachen, dass es der fix für den JS Linter ist